src/main/groovy/wooga/gradle/github/base/GithubBasePlugin.groovy

```diff
@@ -43,54 +44,44 @@ class GithubBasePlugin implements Plugin<Project> {
 
     @Override
     void apply(Project project) {
+        Grgit git = Grgit.init(dir: project.rootProject.projectDir)
```

Comment:
What happens if the project has no git root at the project location? Or some levels up?

Reply:
A git project will be created at `dir`.

Reply:
As we discussed by slack, git-dependent information will now be empty if there is no local git repository available.